synthtool/gcp/templates/node_library/.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,7 +1,10 @@
 Thank you for opening a Pull Request! Before submitting your PR, there are a few things you can do to make sure it goes smoothly:
 - [ ] Make sure to open an issue as a [bug/issue](https://github.com/{{ metadata['repo']['repo'] }}/issues/new/choose) before writing your code!  That way we can discuss the change, evaluate designs, and agree on the general idea
 - [ ] Ensure the tests and linter pass
+- [ ] Ensure your commits follow [Contributing A Patch](https://github.com/{{ metadata['repo']['repo'] }}/blob/master/CONTRIBUTING.md#contributing-a-patch)
 - [ ] Code coverage does not decrease (if any source code was changed)
 - [ ] Appropriate docs were updated (if necessary)
 
 Fixes #<issue_number_goes_here> 🦕
+
+<!-- to update this file visit https://github.com/googleapis/synthtool/edit/master/synthtool/gcp/templates/node_library/.github/PULL_REQUEST_TEMPLATE.md -->
```

Comment:
I believe this would show up in newly opened GitHub issues, so even though it's useful to contributors, I'd rather omit it.

Reply:
If you'll take the checkbox, I'm certainly willing to drop the comment.

Again, my frustration is that the barrier to entry / process of providing fixes is very convoluted. Being repeatedly redirected. For things which aren't particularly obvious. Things which should be stated up front. Things which will generally not change and probably apply to everyone.

I don't mind iterating on large patches. But, the amount of effort required to contribute a very small change shouldn't exceed the amount of effort required to make large changes.